:100644 100644 3e4cd5f... 0000000... M	data/org.pantheon.maya-daemon.desktop
:100644 100644 d95db7c... 0000000... M	data/org.pantheon.maya.desktop.in.in

diff --git a/data/org.pantheon.maya-daemon.desktop b/data/org.pantheon.maya-daemon.desktop
index 3e4cd5f..518e4e5 100644
--- a/data/org.pantheon.maya-daemon.desktop
+++ b/data/org.pantheon.maya-daemon.desktop
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Name=Maya Daemon
 Comment=View Event Reminders
-Icon=office-calendar
+Icon=org.pantheon.maya
 Exec=maya-calendar-daemon
 Terminal=false
 Type=Application
diff --git a/data/org.pantheon.maya.desktop.in.in b/data/org.pantheon.maya.desktop.in.in
index d95db7c..12381f3 100644
--- a/data/org.pantheon.maya.desktop.in.in
+++ b/data/org.pantheon.maya.desktop.in.in
@@ -4,7 +4,7 @@ _Name=@APP_NAME@
 _GenericName=Calendar
 _Comment=View and schedule events
 Exec=maya-calendar %U
-Icon=office-calendar
+Icon=org.pantheon.maya
 Terminal=false
 Categories=GTK;Office;Calendar;
 StartupNotify=true
